content/master/guides/composition-functions.md

```diff
@@ -0,0 +1,1002 @@
+     honored unless the relevant Crossplane feature flag is enabled, and may be
+     changed or removed without notice.
+
+     A Function represents a Composition Function.
+
+FIELDS:
+   config       <>
+     Config is an optional, arbitrary Kubernetes resource (i.e. a resource with
+     an apiVersion and kind) that will be passed to the Composition Function as
+     the 'config' block of its FunctionIO.
+
+   container    <Object>
+     Container configuration of this Function.
+
+   name <string> -required-
+     Name of this Function. Must be unique within its Composition.
+
+   type <string> -required-
+     Type of this Function.
+```
+{{< /expand >}}
+
+{{< expand "An example of most Composition Function configuration options." >}}
+```yaml
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: example
+spec:
+  compositeTypeRef:
+    apiVersion: database.example.org/v1alpha1
+    kind: XPostgreSQLInstance
+  functions:
+  - name: my-cool-Function
+    # Currently only Container is supported. Other types may be added in future.
+    type: Container
+    # Configuration specific to type: Container.
+    container:
+      # The OCI image to pull and run.
+      image: xkpg.io/my-cool-Function:0.1.0
+      # Whether to pull the Function image Never, Always, or IfNotPresent.
+      imagePullPolicy: IfNotPresent
+      # Note that only resource limits are supported - not requests.
+      # The Function will be run with the specified resource limits, specified
+      # in Kubernetes-style resource.Quantity form.
+      resources:
+        limits:
+          # Defaults to 128Mi
+          memory: 64Mi
+          # Defaults to 100m (a 10th of a core)
+          cpu: 250m
+      # Defaults to 'Isolated' - i.e an isolated network namespace with no
+      # network access. Use 'Runner' to allow a Function access to the runner's
+      # (e.g. the xfn container's) network namespace.
+      network:
+        policy: Runner
+      # How long the Function may run before it's killed. Defaults to 20s.
+      # Keep in mind the Function pipeline is typically invoked once every
+      # 30 to 60 seconds.
+      timeout: 30s
+    # An arbitrary Kubernetes resource. Passed to the Function as the config
+    # block of its FunctionIO. Doesn't need to exist as a Custom Resource (CR),
+    # since this resource doesn't exist by itself in the API server but must be
+    # a valid Kubernetes resource (i.e. have an apiVersion and kind).
+    config:
+      apiVersion: database.example.org/v1alpha1
+      kind: Config
+      metadata:
+        name: cloudsql
+      spec:
+        version: POSTGRES_9_6
+```
+{{< /expand >}}
+
+Use `kubectl describe <xr-kind> <xr-name>` to debug Composition Functions. Look
+for status conditions and events. Most Functions will emit events associated
+with the XR if they experience issues.
+
+## Building a Function
+
+ Crossplane is relatively unopinionated about how a Composition Function is
```

Comment:
I'd prefer to keep "unopinionated" here. Anecdotal/subjective, but I feel like it's more common than "doesn't have opinions".

Reply:
Unopinated is a more complex word, enough so that the Chrome spell checker doesn't recognize it. Being straight forward with simple language of "Doesn't have" is more internationally inclusive.